index.html

```diff
@@ -4975,6 +4975,646 @@ <h2 id="AudioWorkerParamDescriptor">
+              </p>
+              <dl class="parameters">
+                <dt>
+                  DOMString name
+                </dt>
+                <dd>
+                  A string key that represents a class definition to be
+                  registered. This key is used to resolve an
+                  <a><code>AudioWorkletProcessor</code></a> during construction
+                  of an <a><code>AudioWorkletNode</code></a>. Throws an
+                  exception if another
+                  <a><code>AudioWorkletProcessor</code></a> has already
+                  registered this key.
+                </dd>
+                <dt>
+                  Function classConstructor
+                </dt>
+                <dd>
+                  A class definition extended from
+                  <a>AudioWorkletProcessor</a>. Throws if this is not a class
+                  definition or it does not extend
+                  <a><code>AudioWorkletProcessor</code></a>.
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+        </section>
+        <section>
+          <h2 id="AudioWorkletProcessor">
+            The <dfn>AudioWorkletProcessor</dfn> interface
+          </h2>
+          <p>
+            This interface represents an audio processing module that runs on
+            the audio <a>rendering thread</a>. It is defined in an
+            <a><code>AudioWorkletGlobalScope</code></a> and the definition of
+            the class manifests the actual audio processing mechanism of a
+            custom audio node. <a><code>AudioWorkletProcessor</code></a> can
+            only be instantiated by the construction of an
+            <a><code>AudioWorkletNode</code></a> instance.
+          </p>
+          <dl title="interface AudioWorkletProcessor" class="idl">
+            <dt>
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
+              parameterDescriptors
+            </dt>
+            <dd>
+              <p>
+                This getter returns an array containing
+                <a>AudioParamDescriptor</a>s, which are descriptors for each
+                controllable parameter on an <a>AudioNode</a>. This static
+                property is referenced in the construction on an
+                <a>AudioWorkletNode</a> to create instances of an
+                <a>AudioParam</a> in the node.
+              </p>
+            </dd>
+            <dt>
+              readonly attribute AudioContextInfo contextInfo
+            </dt>
+            <dd>
+              <p>
+                This getter returns an <a><code>AudioContextInfo</code></a>
+                object that describes the associated <a>AudioContext</a>.
+              </p>
+            </dd>
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The <code>onmessage</code> handler is called whenever the
```

Comment:
Perhaps it would be better to point 'processing model' section? Basically this can only happen before/after the render quantum. So technically the expression 'whenever' is incorrect them.